static/RELEASE_NOTES.md

```diff
@@ -2,7 +2,7 @@
 
 Here you can find description of changes we've built into each release. While we try our best to make each upgrade automatic and as smooth as possible, there may be some cases where you might need to do something to ensure the application works smoothly. So please read through the release highlights!
 
-## 4.1.0-rc.2 (current version)
+## 4.1.0 (current version)
 
 - Change: list views default to a namespace (instead of listing resources from all namespaces)
```

Comment:
Draw more attention to this change? I think it may take some people by surprise.

Where have all my pods gone? Namespaced Kubernetes resources are now initially shown only for the "default" namespace. Use the namespaces selector to add more.

Reply:
Added an upgrade note.

Reply:
![image](https://user-images.githubusercontent.com/1446224/108081836-1379c300-707a-11eb-9699-ede2394460a3.png)

Reply:
Would it make sense to attach images/previews for some new features in "What's new" page? (e.g. to increase user engagement)

Something like:

![image](https://user-images.githubusercontent.com/6377066/108103796-1b922c80-7093-11eb-9909-b103012c5460.png)

Reply:
If VS Code does it, then yes. :)